Source/pack.h

```diff
@@ -13,66 +15,66 @@ namespace devilution {
 
 #pragma pack(push, 1)
 struct PkItemStruct {
-	Uint32 iSeed;
-	Uint16 iCreateInfo;
-	Uint16 idx;
-	Uint8 bId;
-	Uint8 bDur;
-	Uint8 bMDur;
-	Uint8 bCh;
-	Uint8 bMCh;
-	Uint16 wValue;
-	Uint32 dwBuff;
+	int32_t iSeed;

```

Comment:
Why make it signed now?

Reply:
Might be from the order I rand the replace commands in. Could thing you noticed.